client/src/components/PortfolioTable.tsx

```diff
@@ -558,168 +557,183 @@ const PortfolioTableWithoutI18n = React.memo((props: PortfolioTableProps) => {
-                  );
-                })}
-              </tr>
-            ))}
-          </thead>
-          <tbody>
-            {!!table.getRowModel().rows.length &&
-              (activeFilters.rsunitslatestActive || activeFilters.ownernamesActive) && (
-                <tr>
-                  <td
-                    className="filter-table-alert-container"
-                    colSpan={table.getVisibleFlatColumns().length}
-                  >
-                    {activeFilters.rsunitslatestActive && RsUnitsResultAlert}
-                    {activeFilters.ownernamesActive && OwnernamesResultAlert}
-                  </td>
-                </tr>
-              )}
-            {table.getRowModel().rows.map((row, i) => {
-              return (
-                <Fragment key={row.id}>
-                  <tr className={`row-${i % 2 ? "even" : "odd"}`}>
-                    {row.getVisibleCells().map((cell) => {
+    <div
+      id="PortfolioTable"
+      ref={tableRef}
+      className={classnames(hideScrollFade && "hide-scroll-fade", !isVisible && "is-hidden")}
+    >
+      {!isTableLoaded ? (
+        <Loader loading={true} classNames="Loader-table">
+          {data.length > MAX_TABLE_ROWS_PER_PAGE ? (
+            <>
+              <Trans>Loading {data.length} rows</Trans>
+              <br />
+              <Trans>(this may take a while)</Trans>
+            </>
+          ) : (
+            <Trans>Loading</Trans>
+          )}
+        </Loader>
+      ) : (
+        <>
+          <div className="portfolio-table-container">
+            <table>
+              <thead>
+                {table.getHeaderGroups().map((headerGroup) => (
+                  <tr key={headerGroup.id}>
+                    {headerGroup.headers.map((header) => {
                       return (
-                        <td
-                          key={cell.id}
-                          className={`col-${cell.column.id}`}
-                          style={{
-                            width:
-                              cell.column.getSize() !== 0
-                                ? cell.column.getSize() || undefined
-                                : undefined,
-                          }}
+                        <th
+                          key={header.id}
+                          colSpan={header.colSpan}
+                          className={`col-${header.column.id}`}
+                          style={{ minWidth: header.getSize() || undefined }}
                         >
-                          {flexRender(cell.column.columnDef.cell, cell.getContext())}
-                        </td>
+                          {header.isPlaceholder ? null : (
+                            <>
+                              <div
+                                {...{
+                                  className: header.column.getCanSort()
+                                    ? "cursor-pointer select-none"
+                                    : "",
```

Comment:
```suggestion
                                  className: ! header.column.getCanSort()
                                    ?? "cursor-pointer select-none"      
```

Reply:
this whole little chunk was written in a weird spread notation thing. I cleaned that up, and then used `classnames()` with your suggestion (otherwise the `false` wasn't accepted for that prop)